### PR TITLE
Update faq.rst

### PR DESCRIPTION
**Pull request by ahardin-rh**
##### SUMMARY
Minor style edits and typo fixes

##### ISSUE TYPE
- Docs Pull Request

+label: docsite_pr

##### ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- New Module Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: docs/docsite/rst/reference_appendices/faq.rst [hunk 1]
```diff
@@ -150,12 +150,12 @@ How do I handle the package dependencies required by Ansible package dependencie
 +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
 While installing Ansible, sometimes you may encounter errors such as `No package 'libffi' found` or `fatal error: Python.h: No such file or directory`
-These errors are generally caused by the missing packages which are dependencies of the packages required by Ansible.
+These errors are generally caused by the missing packages, which are dependencies of the packages required by Ansible.
 For example, `libffi` package is dependency of `pynacl` and `paramiko` (Ansible -> paramiko -> pynacl -> libffi).
 
-In order to solve these kinds of dependency issue, you may need to install required packages using the OS native package managers (e.g., `yum`, `dnf` or `apt`) or as mentioned in the package installation guide.
+In order to solve these kinds of dependency issues, you maight need to install required packages using the OS native package managers, like `yum`, `dnf`, or `apt`, or as mentioned in the package installation guide.
```
Comment (acozine) [inline docs/docsite/rst/reference_appendices/faq.rst]:
```suggestion
In order to solve these kinds of dependency issues, you might need to install required packages using the OS native package managers, such as `yum`, `dnf`, or `apt`, or as mentioned in the package installation guide.
```

File: docs/docsite/rst/reference_appendices/faq.rst [hunk 2]
```diff
@@ -150,12 +150,12 @@ How do I handle the package dependencies required by Ansible package dependencie
...
 
-Please refer the documentation of the respective package for such dependencies and their installation methods.
+Refer the documentation of the respective package for such dependencies and their installation methods.
```
Comment (acozine) [inline docs/docsite/rst/reference_appendices/faq.rst]:
```suggestion
Refer to the documentation of the respective package for such dependencies and their installation methods.
```

**Review by acozine — COMMENTED** [review]
Thanks @ahardin-rh for tidying up this page! I've made a couple of suggestions to enhance your fixes even more. To incorporate my suggestions, just click `Apply Suggestion` for each one.

File: docs/docsite/rst/reference_appendices/faq.rst [hunk 3]
```diff
@@ -113,7 +113,7 @@ and run Ansible from there.
 
 .. _python_interpreters:
 
-How do I handle python not having a Python interpreter at /usr/bin/python on a remote machine?
+How do I handle Python not having a Python interpreter at /usr/bin/python on a remote machine?
```
Comment (acozine) [inline docs/docsite/rst/reference_appendices/faq.rst]: Hm. This sounds weird to me. How about:

```suggestion
How do I handle not having a Python interpreter at /usr/bin/python on a remote machine?
```

**Comment by ahardin-rh** [comment]
@acozine Updated! Thanks! Sorry for the delay!

**Review by samccann — APPROVED** [review]

**Comment by samccann** [comment]
Thanks @ahardin-rh for the Ansible documentation fix!

**Pull request merged**